mkt/webapps/migrations/0005_auto_20151201_1858.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
No actual db changes in that migration, as no new descriptors/interactives have been added yet. The migration is only here because of the `help_text` changes.